mobile/lib/pages/common/gallery_viewer.page.dart

```diff
@@ -263,10 +263,6 @@ class GalleryViewerPage extends HookConsumerWidget {
     PhotoViewGalleryPageOptions buildAsset(BuildContext context, int index) {
       var newAsset = loadAsset(index);
 
-      WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
If this logic is removed, would any logic in the `currentAssetProvider` won't be using the correct asset when navigating between them?

Reply:
I believe this is handled by the onpagechanged callback:
https://github.com/immich-app/immich/blob/6caa11d079ba3033ae694525d50069ebd0ebf8ab/mobile/lib/pages/common/gallery_viewer.page.dart#L338